spec/lita-reviewme/github_spec.rb

```diff
@@ -0,0 +1,95 @@
+require "spec_helper"
+
+describe Lita::Reviewme::Github do
+  let(:repo) { "gh_user/repo" }
+  let(:id) { "123" }
+  let(:user) { OpenStruct.new(login: 'iamvery') }
+  let(:title) { 'PR Title' }
+  let(:pull_request) { OpenStruct.new(user: user, title: title) }
+  let(:github_comment_template) { ":eyes: %{reviewer}" }
+  let(:github_comment) { true }
+  let(:github_request_review) { false }
+
+  let(:config) do
+    OpenStruct.new(
+      github_access_token: 'access-token',
+      github_comment_template: github_comment_template,
+      github_comment: github_comment,
+      github_request_review: github_request_review
+    )
+  end
+
+  before do
+    allow_any_instance_of(Octokit::Client).to receive(:pull_request)
+      .and_return(pull_request)
+  end
+
+  subject { described_class.new(config, repo, id) }
+
+  describe "ownership" do
+    it "from the pull request" do
+      expect(subject.owner).to eq('iamvery')
+    end
+
+    describe "unexpected error" do
+      class User; def login; raise Octokit::Error; end; end
```

Comment:
Not sure there's any value in squashing this all down to a single line. Not a big deal, but probably not my style fwiw.